cmd/testrunner.go

```diff
@@ -1,21 +1,60 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/packages"
+	"github.com/elastic/elastic-package/internal/testrunner/system"
 )
 
 func setupTestCommand() *cobra.Command {
-	return &cobra.Command{
+	testSystemCmd := &cobra.Command{
+		Use:   "system",
+		Short: "Run system tests",
+		Long:  "Run system tests for a package.",
+		RunE:  testSystemCommandAction,
+	}
+
+	cmd := &cobra.Command{
 		Use:   "test",
 		Short: "Run test suite for the package",
 		Long:  "Use test runners to verify if the package collects logs and metrics properly.",
 		RunE:  testCommandAction,
 	}
+	cmd.PersistentFlags().BoolP("ignoreMissing", "i", false, "ignore missing tests")
+	cmd.AddCommand(
+		testSystemCmd)
+	return cmd
 }
 
-func testCommandAction(cmd *cobra.Command, args []string) error {
-	fmt.Println("test is not implemented yet.")
+func testSystemCommandAction(cmd *cobra.Command, args []string) error {
+	packageRootPath, found, err := packages.FindPackageRoot()
+	if !found {
+		return errors.New("package root not found")
+	}
+	if err != nil {
+		return errors.Wrap(err, "locating package root failed")
+	}
+
+	ignoreMissing, err := cmd.Flags().GetBool("ignoreMissing")
+	if err != nil {
+		return errors.Wrap(err, "error parsing ignoreMissing flag")
+	}
+
+	err = system.Run(packageRootPath)
+	if err != nil && ignoreMissing && err == system.ErrNoSystemTests {
```

Comment:
On further thought I think the the `test*Action` functions here should try to be ignorant of any testing-specific flags such as `--ignoreMissing`. I think they should parse such flags' values into variables and then pass the variables to the test runner for further interpretation. So I'm thinking of moving this `if` condition here inside the  implementation of `system.Run`. 

LMK if you disagree @mtojek.

Reply:
Let's consider the use case:

There is an integration that has system tests, but doesn't have pipeline tests. Pipeline test runner shouldn't fail the `elastic-package test` command.

That's why I was thinking about detecting test directories (package scope, dataset scope) in the `cmd/testrunner.go` and for each of them run all appropriate runners.

Sample flow:

Find test directories:
1. Found `nginx/test`.
2. Found `nginx/access/test`.
3. Found `nginx/error/test`.

Use pipeline test runner:
4. Run pipeline tests for `nginx/access/test`.
5. Run pipeline tests for `nginx/error/test`.

Use system test runner:
6. Run system tests for `nginx/test`.
7. Run system tests for `nginx/access/test`.
8. Run system tests for `nginx/error/test`.

We can sync up/discuss this later on if you prefer.

Reply:
I've implemented this now.